src/dashboard/Data/Browser/EditRowDialog.react.js

```diff
@@ -20,10 +20,10 @@ export default class EditRowDialog extends React.Component {
     super(props);
 
     const { selectedObject } = this.props;
-    const { currentObject, openObjectPickers, expandedTextAreas } = this.initializeState(
+    const { currentObject, openObjectPickers } = this.initializeState(
```

Comment:
These changes also look strange to me

Reply:
these changes do not occur anymore in the latest commit